scripts/paper/generate_stwo_primitive_lookup_vs_naive_figure.py

```
@@ -0,0 +1,229 @@
+  <text x="{WIDTH/2:.1f}" y="{HEIGHT - 48}" text-anchor="middle"
+        font-family="STIX Two Text, Georgia, serif" font-size="24" fill="#6B7280">
+    {escape(footnote)}
+  </text>
+</svg>
+"""
+
+
+def write_optional_rasters(svg_path: Path) -> None:
+    png_path = svg_path.with_suffix(".png")
+    pdf_path = svg_path.with_suffix(".pdf")
+    rsvg = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)